neuraxle/metaopt/auto_ml.py

```diff
@@ -0,0 +1,358 @@
+"""
+Neuraxle's Automatic Machine Learning Classes
+====================================
```

Comment:
The title underline needs to be equal or longer than the title for the title to compile in RST sphinx. Change to: 

```
Neuraxle's Automatic Machine Learning Classes
=================================================
```